screed/screedRecord.py

```diff
@@ -26,6 +26,12 @@ def __getattr__(self, name):
         except KeyError:
             raise AttributeError(name)
 
+    def __len__(self):
+        return len(self.sequence)
+
+    def __len__(self):
+        return len(self.sequence)
+
```

Comment:
Done

Gabriel Pratt
Bioinformatics Graduate Student, Yeo Lab
University of California San Diego

On Mon, Feb 23, 2015 at 1:37 PM, Michael R. Crusoe <notifications@github.com
> wrote:

> In screed/screedRecord.py
> <https://github.com/ged-lab/screed/pull/18#discussion_r25204861>:
>
> > @@ -26,6 +26,12 @@ def __getattr__(self, name):
> >          except KeyError:
> >              raise AttributeError(name)
> >
> > +    def __len__(self):
> > +        return len(self.sequence)
> > +
> > +    def __len__(self):
> > +        return len(self.sequence)
> > +
>
> Merge burp? Double __len__
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ged-lab/screed/pull/18/files#r25204861>.
>